modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -138,8 +138,7 @@ class SeqTranslate(site: Site, systems: Systems, settings: Settings) {
       case _         => false
     }
 
-  def sequence(settings: Settings)
-              (obsId: SPObservationID, sequenceConfig: ConfigSequence, name: String):
+  def sequence(obsId: SPObservationID, sequenceConfig: ConfigSequence, name: String):
```

Comment:
I think what happened was that settings was converted to a class parameter, but not removed from the function.